test/dummy/db/schema.rb

```diff
@@ -0,0 +1,45 @@
+# encoding: UTF-8
```

Comment:
I thought that you intentionally didn't want to track the dummy app db schema in Git . I think that one potential issue with this is when there's a small change in the db adapter (in how it dumps schema) - you'd need to include such changes coming from a dependency (adapter).